lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_object.rb

```diff
@@ -192,11 +192,11 @@ def state_machine?
 
     def attribute_value_to_xml(value, xml)
       case value.class.to_s
-      when 'MiqAePassword'            then xml.Password OPAQUE_PASSWORD
-      when 'String'                   then xml.String   value
-      when 'Fixnum'                   then xml.Fixnum   value
-      when 'Symbol'                   then xml.Symbol   value.to_s
-      when 'TrueClass', 'FalseClass'  then xml.Boolean  value.to_s
+      when 'MiqAePassword'            then xml.Password(OPAQUE_PASSWORD)
+      when 'String'                   then xml.String(  value)
+      when 'Fixnum'                   then xml.Fixnum(  value)
+      when 'Symbol'                   then xml.Symbol(  value.to_s)
+      when 'TrueClass', 'FalseClass'  then xml.Boolean( value.to_s)
```

Comment:
@astrozzc Please cleanup the leading whitespace.

Reply:
The YAML.safe_load changes are removed from this PR. And the leading whitespaces are fixed.